heron/tools/cli/src/python/args.py

```diff
@@ -151,6 +154,32 @@ def add_extra_launch_classpath(parser):
   '''
   parser.add_argument(
       '--extra-launch-classpath',
-      metavar='(a string; additional JVM class path for launching topology)',
-      default="")
+      metavar='CLASS_PATH',
+      default="",
+      help='Additional JVM class path for launching topology')
+  return parser
+
+def add_dry_run(parser):
```

Comment:
this function is used only in [submit.py](https://github.com/twitter/heron/pull/1629/files/cbcf7659a76643643cc625ce0918333a8066b722#diff-c3a12db5e886d4de25105d68c3e96929R56) and [update.py](https://github.com/twitter/heron/pull/1629/files/cbcf7659a76643643cc625ce0918333a8066b722#diff-94fa50525b12c4c96dc2868c1519c079R51). Passing `--dry-run` to other subcommands will get command line parsing errors (assuming that bad `.heronrc` parser does not break things here...)